pennylane/drawer/tape_mpl.py

```diff
@@ -110,6 +110,9 @@ def _tape_mpl(tape, wire_order=None, show_all_wires=False, decimals=None, **kwar
 
     drawer = MPLDrawer(n_layers=n_layers, n_wires=n_wires, wire_options=wire_options)
 
+    if n_wires == 0:
```

Comment:
What case does it cover? A tape that has no wires?

Reply:
Exactly. A tape with only `qml.Snapshot` would fail before because the number of wires cannot be inferred for an "empty" tape. This check just ensures that any numerical errors caused by the tape not having any wires (which is what would happen with `qml.Snapshot`) don't have a chance to occur as the expected figure would be empty either way.